src/features/ajax/instrument/index.js

```diff
@@ -13,18 +13,21 @@ import { wrapFetch, wrapXhr } from '../../../common/wrap'
 import { parseUrl } from '../../../common/url/parse-url'
 import { DT } from './distributed-tracing'
 import { responseSizeFromXhr } from './response-size'
-import { FeatureBase } from '../../../common/util/feature-base'
-import globalScope from '../../../common/util/global-scope'
+import { InstrumentBase } from '../../utils/instrument-base'
+import { FEATURE_NAME } from '../constants'
+import { FEATURE_NAMES } from '../../../loaders/features/features'
+import { globalScope } from '../../../common/util/global-scope'
 
 var handlers = ['load', 'error', 'abort', 'timeout']
 var handlersLen = handlers.length
 
 var origRequest = originals.REQ
 var origXHR = globalScope?.XMLHttpRequest
 
-export class Instrument extends FeatureBase {
-  constructor(agentIdentifier) {
-    super(agentIdentifier)
+export class Instrument extends InstrumentBase {
```

Comment:
InstrumentBase is responsible for importing its matching aggregator file, using `FEATURE_NAME`